helix-core/src/main/java/org/apache/helix/controller/rebalancer/AutoRebalancer.java

```diff
@@ -129,14 +129,12 @@ public IdealState computeNewIdealState(String resourceName,
     ZNRecord newMapping = _rebalanceStrategy
         .computePartitionAssignment(allNodes, liveNodes, currentMapping, clusterData);
 
-    if (LOG.isDebugEnabled()) {
-      LOG.debug("currentMapping: " + currentMapping);
-      LOG.debug("stateCountMap: " + stateCountMap);
-      LOG.debug("liveNodes: " + liveNodes);
-      LOG.debug("allNodes: " + allNodes);
-      LOG.debug("maxPartition: " + maxPartition);
-      LOG.debug("newMapping: " + newMapping);
-    }
+      LOG.debug("currentMapping: {}", currentMapping);
```

Comment:
nit: alignment may be off. please fix it if possible.

Reply:
TFTR. Updated